app/Modules/Group/Events/Traits/IsPublishableApplicationEvent.php

```diff
@@ -18,16 +18,28 @@ public function getEventType(): string
     }
 
     public function getPublishableMessage(): array
-    {
-        return [
-            'expert_panel' => [
-                'id' => $this->group->uuid,
-                'name' => $this->group->displayName,
-                'type' => $this->group->fullType->name,
-                'affiliation_id' => $this->group->expertPanel->affiliation_id
-            ],
-        ];
-    }
+{
+    return [
+        'expert_panel' => array_merge([
+            'id' => $this->group->uuid,
+            'long_name' => $this->group->name,
+            'short_name' => $this->group->expertPanel->short_base_name,
+            'status' => optional($this->group->groupStatus)->name, // Retrieve the status name
+            'group updated date' => $this->group->updated_at,
+            'parent_group' => optional($this->group->parentGroup)->name,
+            'type' => $this->group->fullType->name,
+            'affiliation_id' => $this->group->expertPanel->affiliation_id,
+            'scope description' => $this->group->expertPanel->scope_description,
```

Comment:
Again, uses different convention for separator (space instead of underscore). Noting that this may make processing in json more difficult by web team (since javascript can treat my_object['attribute'] as the same as my_object.attribute (but only if the name "attribute" has no spaces...)

Reply:
addressed in later commit